src/drawers/arcDrawer.ts

```diff
@@ -43,6 +43,11 @@ export module _Drawer {
                           .value(valueAccessor)(data);
 
       drawSteps.forEach(s => delete s.attrToProjector["value"]);
+      pie.forEach((slice) => {
+        if (slice.value < 0) {
+          _Util.Methods.warn("Negative values will not render correctly in a pie chart.");
+        }
+      });
```

Comment:
This is fine for now.  Just noting that we will want to handle these types of verification checks in a more first-class way.  This and the similar answers we have are not ideal.